jina/types/arrays/neural_ops.py

```diff
@@ -7,6 +7,8 @@
 from ...importer import ImportExtensions
 from ...math.helper import top_k, minmax_normalize
 
+from jina.logging.predefined import default_logger
```

Comment:
do relative import from jina as in the rest of the file

Reply:
done

Reply:
I do not see it done